content/en/dora_metrics/deployments/apm.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Configuring APM Deployment Tracking for DORA Metrics
+kind: documentation
+description: Learn how to configure APM Deployment Tracking as a data source for DORA Metrics deployments.
+aliases:
+- /continuous_integration/dora_metrics/setup/apm
+is_beta: true
+further_reading:
+- link: "https://app.datadoghq.com/release-notes?category=Software%20Delivery"
+  tag: "Release Notes"
+  text: "Check out the latest Software Delivery releases! (App login required)"
+- link: "/continuous_integration/dora_metrics/setup/incidents"
+  tag: "Documentation"
+  text: "Learn about sending incident events"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "https://github.com/DataDog/datadog-ci"
+  tag: "Source Code"
+  text: "Learn about the datadog-ci CLI tool"
+- link: "/continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn about Deployment Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/Eqq6uXfGjYxmqpjDA" header="false" >}}
+The DORA Metrics private beta is closed. Fill out the form below to be added to the waitlist.
+{{< /callout >}}
+
+## Overview
+APM [Deployment Tracking][2] can be configured as a data source for deployments in DORA Metrics.
+
+For service deployments tracked by APM to contribute to DORA Metrics, the following requirements must be met:
+- Your service has [metadata][1] defined in the Service Catalog.
+- Your service has [unified service tagging][3] enabled. Deployments are identified via the `version` tag.
+
+## Change lead time
+For service deployments tracked by APM to contribute to Change Lead Time, the following requirement must be met:
+- Your application telemetry is tagged with git information. You can enable this [in APM][4] or see [documentation][5].
+- Your repository metadata is synchronized to Datadog via the GitHub integration.
+
+For deployments identified via Deployment Tracking, change lead time is computed from time of first commit creation to when that commit is first seen in a new version.
+
+### Breakdown metrics
```

Comment:
I think we shouldn't duplicate this section here.

Reply:
I duplicated it because we don't have a `dora.deploy_time` metric for APM deployments (and customers onboarding to DORA with this data source for the first time wouldn't have the expectation that there is a `dora.deploy_time` metric, so it's not a limitation)